projects/canopy/src/lib/forms/sort-code/sort-code.component.scss

```diff
@@ -0,0 +1,15 @@
+.lg-sort-code-field {
+  display: block;
+  margin-bottom: var(--component-margin);
+  line-height: var(--line-height-sm);
+}
+
+.lg-sort-code__fields {
+  display: flex;
+}
+
+.lg-sort-code__input {
+  flex-direction: column;
+  margin-right: var(--space-sm);
+  width: 3.75rem; // 60px
```

Comment:
I'm wondering if we should introduce a `mixin` to translate `px` into `rem`. Maybe it would make it easier to understand without having to add the comment of the corresponding `px` value?

Reply:
That could be quite nice, it also saves you from having to do the calculation each time. I'm happy to add one in this PR and use it in this instance, and we could have an issue open to clean up the rest of the values (which I'd be happy to do also 🙂 ). I'll check with the others in Slack using poll to see if the majority agrees.

Reply:
I need to have a think about this, I guess the designs are in pixels and thats where the pain comes from. Is the real fix to get the designs also done in rem/em. Or just switch to pixels, I think it needs a bit of reading up.

Reply:
Yes, at the end of the day `rem` is mostly useful when applying font sizes. Not 100% sure we need to use it for width/height.